lib/creditsafe/request/find_company.rb

```diff
@@ -35,15 +46,29 @@ def message
         }
       end
 
-      private
+      # rubocop:disable Style/CyclomaticComplexity, Metrics/AbcSize
```

Comment:
Potentially controversial, but I think this method makes sense doing the job that it's doing. I could potentially split it down more if you feel this makes sense.